spec/factories/facility_access.rb

```diff
@@ -30,4 +30,1250 @@
     source_updated '2017-03-31T00:00:00'
     local_updated  '2017-06-12T21:04:58Z'
   end
+
+  factory :vha_648A4, class: Facilities::VHAFacility do
```

Comment:
NITPICK: It's unclear why this is called "facility_access.rb"

factoriy files are generally pluralized, particularly when they correspond to ActiveRecord or ActiveModel objects. Since right now this is shared with the existing code we can't tackle it, but perhaps there will be some opportunity to clean this up in the future? A github issue to track might suffice for now.

Reply:
I was able to split my new factories out into appropriately named files.. much better